doc/news/_preparation_next_release.md

```diff
@@ -167,6 +167,10 @@ plugins, bindings and tools are always up to date. Furthermore, we changed:
 These notes are of interest for people developing Elektra:
 
 - A build job checks if PRs modify the release notes. *(Markus Raab)*
+- `clang` tests have been ported to the new build system *(Lukas Winkler et al)*
+- Several improvments to the build system have been implemented *(Lukas Winkler)*:
+  - Better Docker image handling
+  - abort of previously queued but unfinished runs on new commits
```

Comment:
GTEST_ROOT not mentioned

Reply:
?

Reply:
?

Reply:
I added the entry hence I do not understand your question?

Reply:
Maybe the github view is buggy, I only see following changes in doc/news/_preparation_next_release.md:

```
+- `clang` tests have been ported to the new build system *(Lukas Winkler et al)*
+- Several improvments to the build system have been implemented *(Lukas Winkler)*:
+  - Better Docker image handling
+  - abort of previously queued but unfinished runs on new commits
...
+- [Google Test][] is installed in Docker images used by the build system.
```

Which is really little compared to what you actually did.

Reply:
There is a seperate entry by @sanssecours as this was his addition. I only added it (as the comment says) to the build system so it does not get redownloaded every run.

There was also no entry that removed GTEST_ROOT as a env variable so I did not mention it in the entry as it would certainly only confuse people that were relying on it.

Reply:
> There was also no entry that removed GTEST_ROOT as a env variable

Afaik this (GTEST_ROOT accepted as env variable) was never the case.

This (#2003) is the first PR that changes the behavior that GTEST_ROOT can be an env variable. So it needs to be documented in the section "Notes for Maintainer" within "doc/news/_preparation_next_release.md".

If you need help please say so. Writing this line is easier than discussing it :smile: 

I am open for improvements of how we write "doc/news/_preparation_next_release.md". That the release notes is updated within every PR is a new feature. Obviously it has disadvantages: people not used to writing release notes suddenly need to write parts of it. Such discussions in a separate issue, though.